### PR TITLE
Create linked-list-cycle.py

### PR DESCRIPTION
**Pull request by manrajg13**
## Problem(s) Solved

- Linked List Cycle

### CODE REVIEW

File: manraj/linked-list-cycle.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Solution:
+    def hasCycle(self, head: Optional[ListNode]) -> bool:
+        while head:
+            if head.val == "seen":
+                return True
+            head.val = "seen"
```
Comment (zainaraza43) [inline manraj/linked-list-cycle.py:6]: thIs is fucking genius LMFAOOOO

**Review by zainaraza43 — APPROVED** [review]
LGTM 🚀

**Review by prairir — COMMENTED** [review]
:pray: :bow: LGTM :rocket: :rocket:

File: manraj/linked-list-cycle.py (hunk 1 above)
Comment (manrajg13) [reply manraj/linked-list-cycle.py:6]: they let me cook I'm afraid

**Pull request merged**